bot.py

```diff
@@ -106,6 +106,21 @@ async def issues(ctx, repoName=None, state='open'): # !git issues MLH-Fellowship
 		for i in issues:
 			await ctx.send('Issue Title: ' + i.title + '\nIssue Number: ' + str(i.number) +'\nIssue Link: https://github.com/' + repo.name + '/issues/' + str(i.number))
 
+@bot.command(brief='displays recent issues')
+async def recent_issues(ctx, repoName=None, state='open'): # !git recent_issues MLH-Fellowship/github-discord-bot open
```

Comment:
looks good to me! One thing we could add is make the maxlength an argument, so the user can decide how many issues they want to see (5,10,...)